clients/cloud/nodejs/consumer.js

```diff
@@ -0,0 +1,75 @@
+/* Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * =============================================================================
+ *
+ * Consume messages from Confluent Cloud
+ * Using the node-rdkafka client for Apache Kafka
+ *
+ * =============================================================================
+ */
+
+const Kafka = require('node-rdkafka');
+const { configFromCli } = require('./config');
+
+function createConsumer(config, onData) {
+  const consumer = new Kafka.KafkaConsumer({
+    'metadata.broker.list': config['bootstrap.servers'],
```

Comment:
use 'bootstrap.servers' for consistency. 'metadata.broker.list' is the legacy property name.